services/dsp-a/src/public/js/bidding-logic.js

```diff
@@ -0,0 +1,38 @@
+/*
+ Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+function generateBid(
+  interestGroup,
+  auctionSignals,
+  perBuyerSignals,
+  trustedBiddingSignals,
+  browserSignals,
+) {
+  return {
+    bid: Math.floor(Math.random() * 100, 10),
```

Comment:
Should we add a little comment explaining that the price is picked randomly for demo purpose?

Reply:
Good call